Pinta.Docking/Pinta.Docking.csproj

```diff
@@ -1,17 +1,110 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
-    <Nullable>enable</Nullable>
 
-    <RootNamespace>Pinta.Docking</RootNamespace>
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
```

Comment:
Should this line be removed? I think it conflicts with the Library output type below.